app/controllers/admin/npq/applications/eligible_for_funding_controller.rb

```diff
@@ -22,9 +22,9 @@ def update
 
           set_eligibility_status(eligible_for_funding_params["eligible_for_funding"])
           name = @npq_application.participant_identity.user.full_name
+          @npq_application.current_user = current_user
```

Comment:
I would suggest moving the logic of updating the eligibility for funding into a service where we can add all the current logic and avoid the callbacks.

If not maybe we can do this here:
```
@npq_application.updated_by = current_user.email
@npq_application.eligible_for_funding_updated_at = Time.zone.now
```
to avoid adding a callback?